src/common/components/private-repos-info/index.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import url from 'url';
+import fetchJsonp from 'fetch-jsonp';
+
+import Popover from '../popover';
+import Button from '../vanilla/button';
+
+import styles from './private-repos-info.css';
+
+const MAILCHIMP_FORM_URL = 'https://canonical.us3.list-manage.com/subscribe/post-json';
+const MAILCHIMP_FORM_U = '56dac47c206ba0f58ec25f314';
+const MAILCHIMP_FORM_ID = '381f5c55f1';
+
+export default class PrivateReposInfo extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      showPopover: false,
+      popoverOffsetLeft: 0,
+      popoverOffsetTop: 0,
+      subscribeEmail: '',
+      subscribeSuccess: false,
+      subscribeError: false,
+      message: ''
+    };
+  }
+
+  componentDidMount() {
+    this.onBoundDocumentClick = this.onDocumentClick.bind(this);
+    document.addEventListener('click', this.onBoundDocumentClick);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('click', this.onBoundDocumentClick);
+  }
+
+  onDocumentClick() {
+    this.setState({
+      showPopover: false
+    });
+  }
+
+  onHelpClick(event) {
+    // prevent help click from triggering document click
+    event.nativeEvent.stopImmediatePropagation();
+
+    const { target } = event;
+
+    this.setState({
+      showPopover: !this.state.showPopover,
+      popoverOffsetLeft: target.offsetLeft + (target.offsetWidth / 2),
+      popoverOffsetTop: target.offsetTop + target.offsetHeight
```

Comment:
Positioning popover centered under clicked element.
I wonder whether it should be calculated here or maybe it suits more in Popover itself.
But it was just easier to pass these as a props to Popover instead of passing whole target element and letting Popover decide how to calculate the positioning.